inmap/cmd/config.go

```diff
@@ -118,8 +118,8 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 			"there has been an error parsing the configuration file: %v\n", err)
 	}
 
-	for i, v := range config.OutputVariables {
-		config.OutputVariables[i] = os.ExpandEnv(v)
+	for k, v := range config.OutputVariables {
+		config.OutputVariables[k] = os.ExpandEnv(v)
```

Comment:
You can see the documentation for os.ExpandEnv [here](https://golang.org/pkg/os/#ExpandEnv). Basically, it allows the variables and variable names to include environment variables. This allows the user a little more flexibility in how they run the model.